hermes/Earthfile

```diff
@@ -4,39 +4,39 @@ VERSION 0.7
 
 # Set up our target toolchains, and copy our files.
 builder:
-    FROM github.com/input-output-hk/catalyst-ci/earthly/rust:v2.0.12+rust-base
+    FROM github.com/input-output-hk/catalyst-ci/earthly/rust:v2.0.13+rust-base
```

Comment:
You shouldn't be modifying anything to do with the base hermes Earthfile, or cardano-chain-follower in the PR.